js/models/base.js

```diff
@@ -268,7 +269,7 @@ export class List extends Base {
     }
 
     on_fetched(data) {
-        this.items = data.obj;
+        this.items = this.model ? data.obj.map(o => new this.model({data: o})) : data.obj;
```

Comment:
this looks like some black magic code having a wider scope than the PR desc

Reply:
Details on how and why added in the description